autometa/taxonomy/ncbi.py

```diff
@@ -299,23 +284,15 @@ def parse_names(self):
 
     # @timeit
     def parse_nodes(self):
-        """Short summary.
-
-        Parameters
-        ----------
-        verbose : boolean
-            prints progress to terminal (the default is False).
+        """
+        Parses through `nodes.dmp` database and returns a nested dictionary with child taxid as keys
+        and corresponding parent taxid and rank as items 
```

Comment:
The ReturnType is not necessary here as this should be described under `Returns`

You could also cross-reference other functions that use the generated data structure. Similarly you could note that this is run when a new instance is constructed.

```suggestion
        Parse the `nodes.dmp` database to be used later by :func:`~ncbi.NCBI.parent`, :func:`~ncbi.NCBI.lineage`
```

```suggestion
        Parse the `nodes.dmp` database.
        
        Note: This is performed when a new NCBI class instance is constructed.
```